test/system/test_cvp_client_api.py

```diff
@@ -808,53 +808,34 @@ def test_api_get_configlets_and_mappers(self):
     def test_api_get_configlet_builder(self):
         ''' Verify get_configlet_builder
         '''
-        try:
-            # Configlet Builder for pre 2019.x
-            cfglt = self.api.get_configlet_by_name('SYS_TelemetryBuilderV2')
-        except CvpApiError as e:
-            if 'Entity does not exist' in e.msg:
-                # Configlet Builder for 2019.x
-                try:
-                    cfglt = self.api.get_configlet_by_name(
-                        'SYS_TelemetryBuilderV3')
-                except CvpApiError as e:
-                    if 'Entity does not exist' in e.msg:
-                        # Configlet Builder for 2021.x - 2022.1.1
-                        try:
-                            cfglt = self.api.get_configlet_by_name(
-                                'SYS_TelemetryBuilderV4')
-                        except CvpApiError as e:
-                            if 'Entity does not exist' in e.msg:
-                                # Configlet Builder for 2022.2.0 - 2022.3.X
-                                try:
-                                    cfglt = self.api.get_configlet_by_name(
-                                        'SYS_TelemetryBuilderV5')
-                                except CvpApiError as e:
-                                    if 'Entity does not exist' in e.msg:
-                                        # Configlet Builder for 2022.3.X+
-                                        cfglt = self.api.get_configlet_by_name(
-                                            'SYS_TelemetryBuilderV6')
-                    else:
-                        raise
-            else:
-                raise
-        result = self.api.get_configlet_builder(cfglt['key'])
+        cfglt_bldr_key = None
```

Comment:
After testing with the develop version, I don't think we need these changes any more.

Reply:
Keeping this update for simplification and better format for going foward.

Reply:
Discussed that it is best if that this change remainss.